pkg/hooks/launch.go

```diff
@@ -620,12 +621,12 @@ func parseDockerCommand(dockerCmd string) (string, string, error) {
 	return containerName, networkName, nil
 }
 
-func getInodeNumber(pid int) uint64 {
-
+func (h *Hook) getInodeNumber(pid int) uint64 {
 	filepath := filepath.Join("/proc", strconv.Itoa(pid), "ns", "pid")
```

Comment:
any reason to move `getInodeNumber` into hooks?

Reply:
Because I want to use `h.logger.Error` in this function.

Reply:
You could rather pass it as a parameter

Reply:
ok!
I fixed it! [pass logger as a parameter](https://github.com/keploy/keploy/pull/1351/commits/7e75d9266f95ddc90c59a7b972e436eaa43dc218)